apps/user/src/components/form/지원자정보/ProfileUpload/ProfileUpload.tsx

```diff
@@ -0,0 +1,97 @@
+import { color, font } from '@maru/theme';
+import { Button, Column } from '@maru/ui';
+import {
+    useImageFileChange,
+    useImageFileDragAndDrop,
+    useOpenUploadImageFile,
+    useUploadProfileImageFile,
+} from './ProfileUpload.hooks';
+import styled from 'styled-components';
+import { useState } from 'react';
+
+const ProfileUpload = () => {
+    const [profileImage, setProfileImage] = useState('');
+
+    const { uploadProfileImageFile } = useUploadProfileImageFile(setProfileImage);
+    const { imageFileInputRef, handleImageUploadButtonClick } = useOpenUploadImageFile();
+    const { isDragging, onDragEnter, onDragLeave, onDragOver, onDrop } =
+        useImageFileDragAndDrop(uploadProfileImageFile);
+    const { handleImageFileChange } = useImageFileChange(uploadProfileImageFile);
+
+    return (
+        <StyledProfileUpload>
+            <Title>증명사진</Title>
+            {profileImage ? (
+                <ImagePreview src={profileImage} alt="profile-image" />
+            ) : (
+                <ImageUploadBox
+                    onDragEnter={onDragEnter}
+                    onDragLeave={onDragLeave}
+                    onDragOver={onDragOver}
+                    onDrop={onDrop}
+                    isDragging={isDragging}>
+                    <Column gap={12} alignItems="center">
+                        <Button size="SMALL" onClick={handleImageUploadButtonClick}>
+                            파일 선택
+                        </Button>
+                        <ImageUploadText>또는</ImageUploadText>
+                        <ImageUploadText>여기로 사진을 끌어오세요</ImageUploadText>
+                    </Column>
+                </ImageUploadBox>
+            )}
+            {profileImage.length !== 0 && (
```

Comment:
24번째 줄이랑 똑같이

```suggestion
            {profileImage && (
```
이렇게 바꿔도 될 것 같아요

Reply:
오 기본값이 ''이라 true로 처리될줄알았는데 아니였군요 ! 세세하게 봐주셔서 감사합니당~